solr/contrib/extraction/src/test-files/extraction/solr/collection1/conf/solrconfig.xml

```diff
@@ -43,6 +43,25 @@
   </updateHandler>
 
 
+  <updateProcessor class="solr.ParseDateFieldUpdateProcessorFactory" name="parse-date">
+    <arr name="format">
+      <str>yyyy-MM-dd['T'[HH:mm[:ss[.SSS]][z</str>
+      <str>yyyy-MM-dd['T'[HH:mm[:ss[,SSS]][z</str>
+      <str>yyyy-MM-dd HH:mm[:ss[.SSS]][z</str>
+      <str>yyyy-MM-dd HH:mm[:ss[,SSS]][z</str>
+      <str>EEE MMM d [HH:mm:ss ][z ]yyyy</str>
+      <str>EEEE, dd-MMM-yy HH:mm:ss [z</str>
+      <str>[EEE, ]dd MMM yyyy HH:mm[:ss] [z</str>
```

Comment:
> BTW I was looking at DateTimeFormatter.RFC_1123_DATE_TIME and noticed the leading day of week is optional.

I also had a little peek and noticed the second of the minute is also optional, here is the code snippet that initializes the default RFC_1123_DATE_TIME DateTimeFromatter:

`.optionalStart()
  .appendLiteral(':')
  .appendValue(SECOND_OF_MINUTE, 2)
  .optionalEnd()`